components/project-card/meta/get-involved.jsx

```diff
@@ -62,8 +72,9 @@ class GetInvolved extends React.Component {
       return null;
 
     return (
-      <div className="get-involved pb-3 mb-3">
-        <h2>Help needed</h2>
+      
```

Comment:
Extra line here.

Note that prettier/eslint is set up on this project but I made it to only raise warnings instead of errors otherwise it will make code review hard (hard to distinguish code changes vs linting fixes). I'm planning to fix all linting errors in the codebase once major updates to Pulse are done. In the meantime... it'd be awesome if we can make sure the new code we are adding to the codebase is lint error free! 😉